data/src/main/java/ca/bc/gov/data/local/entity/TestResultEntity.kt

```diff
@@ -29,5 +30,7 @@ data class TestResultEntity(
     @ColumnInfo(name = "patient_id")
     val patientId: Long,
     @ColumnInfo(name = "collection_date")
-    val collectionDate: Instant
+    val collectionDate: Instant,
+    @ColumnInfo(name = "data_source", defaultValue = "")
```

Comment:
Please mention the defaultvalue = "2". The value of DataSource.PUBLIC_API needs to placed here instead of empty string so that existing records are also updated.